mxcubecore/HardwareObjects/abstract/AbstractAperture.py

```diff
@@ -171,15 +181,20 @@ def set_out(self):
         pass
 
     def is_out(self):
-        """
+        """Is aperture out
+
         Returns:
             bool: True if aperture is in the beam, otherwise returns false
         """
         pass
 
     def force_emit_signals(self):
-        """
-        Reemits all signals
+        """Reemits all signals
```

Comment:
I would stick with the indicative mood everywhere:

```suggestion
        """Reemit all signals.
```